scripts/src/public/services/offer_contents.ts

```diff
@@ -153,17 +153,16 @@ export async function sumCorrectQuizAnswers(offerContent: db.OfferContent, form:
 	}
 
 	const quiz: Quiz = JSON.parse(translatedContent || offerContent.content);  // this might fail if not valid json without replaceTemplateVars
-	let amountSum = 0;
+	// let amountSum = 0;
 
-	for (const page of quiz.pages) {
+	const amountSum = quiz.pages.reduce((sum: number, page: QuizPage | SuccessBasedThankYouPage): number => {
```

Comment:
I don't mind this change but I'm pretty sure `for-of` is more performant than `Array.prototype.reduce` and I'm really not sure this is more readable. What was your rational for making the change?